elasticsearch.md

```diff
@@ -920,6 +920,76 @@ After adding the `filter` rule \(using the block duplication strategy\).
       kibana_index: ".kibana_@{user}"
 ```
 
+#### `response_fields`
+
+This rule allows filtering Elasticsearch responses using defined list of fields. It works in very similar way to `fields` rule. However in contrast to `fields` rule, which filters out document fields, this rule filters out response fields. It **doesn't make use of Field Level Security \(FLS\)** and could be applied to every response returned by Elasticsearch.
+
+It could be configured in two modes:
+  * *whitelist* passing only defined fields
+  * *blacklist* filtering out defined fields.
+
+**Blacklist mode**
+
+Specifies which fields should be filtered out with `~` (other fields from mapping become allowed implicitly). Example:
```

Comment:
filtered out by adding the `~` prefix to the field name. Other fields in the response will be implicitly allowed. For example: